cmd/inspect.go

```diff
@@ -92,6 +92,25 @@ func (cli *CLI) inspect(opts Options, args []string) int {
 	cli.formatter.Print(issues, nil, cli.sources)
 
 	if len(issues) > 0 && !force {
+		if opts.MinimumSeverity != "" {
+			var minSeverity tflint.Severity
+			switch strings.ToLower(opts.MinimumSeverity) {
+			case "warning":
+				minSeverity = 1
+			case "error":
+				minSeverity = 2
+			default:
+				return ExitCodeIssuesFound
+			}
+
+			for _, i := range issues {
+				if i.Rule.Severity() >= minSeverity {
```

Comment:
Well they are defined in order, just arguably backwards, and definitely not the way this code expects:

https://github.com/terraform-linters/tflint-plugin-sdk/blob/3721c4b92974d442bb3b0985f327b1b929a0e579/tflint/issue.go#L4-L13

Error is 0 and Notice is 2.

They're also re-declared which should be resolved:

https://github.com/terraform-linters/tflint/blob/7b4785d2a7515d536c8faf2bb16096829414b1dd/tflint/issue.go#L24-L31

Reply:
Ah I made a hasty assumption here, my bad! I do think it would make more sense to have them in an order from info to error, which would allow for these kinds of comparisons. For now I've implemented a function to convert them to the values I need, but it of course necessitates changing that method if you were to add new severities... But in general it looks like sweeping changes would be needed to add new severities, since there are switch statements using them all over the codebase.
I also removed the redeclared severity constant and references the SDK everywhere instead. Let me know if this is something that should be saved for another PR 🤪